alchemist/alchemist-cognitive-agents/src/main/kotlin/it/unibo/alchemist/model/implementations/actions/Flee.kt

```diff
@@ -10,18 +12,28 @@ import it.unibo.alchemist.model.interfaces.Reaction
  *
  * @param env
  *          the environment inside which the pedestrian moves.
+ * @param reaction
+ *          the reaction which executes this action.
  * @param pedestrian
  *          the owner of this action.
  * @param coords
  *          the coordinates of the position the pedestrian moves away.
  */
 open class Flee<T, P : AbstractEuclideanPosition<P>>(
```

Comment:
To we need this specificity? Can't we set a bound on `Vector<P`> and `Position<P>`?